src/content/en/showcase/2017/with.in.md

```diff
@@ -0,0 +1,331 @@
+
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td>Resolution Independent
+   </td>
+   <td>Typographic Features
+   </td>
+   <td>Performance
+   </td>
+   <td>Ease of Implementation
+   </td>
+  </tr>
+  <tr>
+   <td><a href="https://vr.with.in/archive/text-2d-canvas/">2D Canvas Text</a>
+   </td>
+   <td>
+   </td>
+   <td>X
+   </td>
+   <td>X
+   </td>
+   <td>X
+   </td>
+  </tr>
+  <tr>
+   <td><a href="https://vr.with.in/archive/text-triangulated-vector/">Triangulated Vector Text</a>
+   </td>
+   <td>X
+   </td>
+   <td>
+   </td>
+   <td>X
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td><a href="https://threejs.org/examples/webgl_geometry_text.html">Extruded 3D Text</a>
+   </td>
+   <td>X
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td><a href="https://vr.with.in/archive/text-sdf-bitmap/">Signed Distance Field Bitmap Text</a>
+   </td>
+   <td>X
+   </td>
+   <td>X
+   </td>
+   <td>X
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+## Our Decision: SDF Bitmap Font
+
+2D canvas with `ctx.fillText()` can do text wrapping, letter spacing and line height, but overflow gets cut off, and text will be blurry if you zoom in really far. You could increase the size of the canvas texture, but might hit an upper limit in texture size or performance could suffer if the texture is too big.
+
+Extruded 3D text is essentially the same as triangulated vector text, but with depth and possibly a bevel, so it has at least twice as much geometry. Either of these could work in small doses for titles or logos, but wouldn't perform as well for large quantities of text, and neither has typographic features.
+
+Bitmap fonts use one quad (two triangles) per character, so it uses less geometry and performs better than [triangulated vectors](https://vr.with.in/archive/text-triangulated-vector/). They're still raster based since they use a texture map sprite, but with an SDF shader, they're basically resolution independent, so they look nicer than a 2D canvas texture. Matt DesLauriers' three-bmfont-text also includes reliable typographic features for text wrapping, letter spacing, line height and alignment. Overflow doesn't get cut off. Font size is controlled through scale. We chose this route because it gave us the best options for design while still staying performant. Unfortunately, it wasn't as easy to implement so we'll go through the steps in hopes to alleviate fellow developers working in WebVR.
+
+[mattdesl.svbtle.com/material-design-on-the-gpu](https://mattdesl.svbtle.com/material-design-on-the-gpu)
+
+[mattdesl.github.io/physical-text/index.html](http://mattdesl.github.io/physical-text/index.html)
+
+[github.com/mattdesl/physical-text](https://github.com/mattdesl/physical-text)
+
+
+<figure>
```

Comment:
Maybe float this right beside the main content? It's relatively small so it doesn't take up the full width